Test/prover/z3-hard-timeout.bpl

```diff
@@ -0,0 +1,11 @@
+// RUN: %boogie /proverOpt:C:"-T:1" /proverOpt:BATCH_MODE=true /trace "%s" > "%t"
```

Comment:
Is `T` a timeout option? Might be nice to use the unabbreviated version for readability.

Reply:
Yeah, it is a timeout option. I'm not sure it has a long form, actually. There is a `timeout` parameter, but I _think_ that corresponds to the "soft" timeout set with `-t` (lowercase).